### ISSUE TITLE
[ENH] AutoETS

### ISSUE THREAD

**Issue by TonyBagnall** [aeon-toolkit/aeon#2411]
### Describe the feature or idea you want to propose

Our new experimental forecasting module starts with a really fast ETS implementation. The next stage is AutoETS. But how to search the parameter space? There are many alternatives, grid search, nelder mead, stochastic gradient descent etc. It would be really good to implement this in a configurable way. Feel free to come up with alternative heuristic search algorithms on this thread

### Describe your proposed solution

we need to think about how to design this efficiently, perhaps taking inspiration from scikit-learn?

**Comment by TonyBagnall** [comment]
@aeon-actions-bot assign @alexbanwell1

**Comment by SebastianSchmidl** [comment]
Bayesian optimization using https://optuna.readthedocs.io/en/stable/?

**Comment by SalmanDeveloperz** [comment]
I think no one is working on this issue. I would like to work on this issue.

**Comment by alexbanwell1** [comment]
Hi SalmanDeveloperz,
I have an implementation of this that I was working on in my fork. I shall try and submit a pull request for what I've got this week, but very happy to take suggestions for how it could be improved!
Thanks,
Alex

**Comment by alexbanwell1** [comment]
This has been completed and is now in
